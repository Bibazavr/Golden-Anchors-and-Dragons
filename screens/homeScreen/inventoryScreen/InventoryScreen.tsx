import * as React from "react";
import {StyleSheet, View} from "react-native";

import {BasicSlot} from "./utils/Modal";


export const InventoryScreen = () => {
    return (
        <View style={stylesInventoryScreen.container}>
            <BasicSlot/>
            <View style={stylesInventoryScreen.middle}>
                <BasicSlot/>
                <BasicSlot/>
                <BasicSlot/>
            </View>
            <BasicSlot/>
            <BasicSlot/>
            <View style={stylesInventoryScreen.middle}>
                <BasicSlot/>
                <BasicSlot/>
            </View>
        </View>
    )
}

const stylesInventoryScreen = StyleSheet.create({
    container: {
        flex: 1,
        alignItems: "center",
        justifyContent: "center",
        backgroundColor: '#fff',
    },
    middle: {
        flexDirection: "row",
        justifyContent: "space-around"
    }
});
