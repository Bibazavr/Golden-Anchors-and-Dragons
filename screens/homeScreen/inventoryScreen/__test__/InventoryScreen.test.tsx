import * as React from "react";
import {render} from "@testing-library/react-native";

import {InventoryScreen} from "../InventoryScreen";


it(`renders correctly`, () => {
    const tree = render(<InventoryScreen/>).asJSON();

    expect(tree).toMatchSnapshot();
});