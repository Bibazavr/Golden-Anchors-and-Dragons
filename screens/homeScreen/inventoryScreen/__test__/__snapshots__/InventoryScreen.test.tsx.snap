// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      }
    }
  >
    <Text>
      Сюда надо Запихнуть инвентарь
    </Text>
  </View>
</View>
`;
